### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by alex-grover**
- Add `chore` change type
- Enhance descriptions to make it more clear PR authors are expected to change the template
- Make everything comments by default so instructions to authors are not left in the PR description

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 
-Please include a summary of the changes, including screenshots, if applicable. Please also provide any motivation and context necessary to review these changes. List any dependencies that are required for this change.
+<!-- Please include a summary of the changes, including screenshots, if applicable. Please also provide any motivation and context necessary to review these changes. List any dependencies that are required for this change. -->
 
 ## Type of change
 
-Please delete options that are not relevant.
+<!-- Please delete options that are not relevant.
 
+- [ ] Chore (non-breaking change which has no end user impact)
```
Comment (carlospliego) [inline pull_request_template.md:9]: nice

**Review by carlospliego — APPROVED** [review]

**Comment by bgarzonDarwin** [comment]
@alex-grover I don't know if it could be valuable, but we can use https://github.com/commitizen/cz-cli to enforce the commit style; I believe this could help us have the same set of rules when everyone creates a new PR with the right message and prefix depending on the nature of the changes.

File: pull_request_template.md [hunk 2]
```diff
@@ -1,29 +1,34 @@
...
```
Comment (EnriqueVidal) [inline pull_request_template.md:9]: can we have a `Docs` one too

**Review by EnriqueVidal — APPROVED** [review]

File: pull_request_template.md (hunk 2 above)
Comment (alex-grover) [reply pull_request_template.md:9]: oops auto merge got me, but i was assuming that docs updates would fall under chore as well.

**Comment by alex-grover** [comment]
@bgarzonDarwin that's a good idea. I'm thinking for now we can just use the template so people can opt out at their discretion, but as we grow if that's not enough then we can enforce via that check. How does that sound to you?

**Pull request merged**